tsfc/driver.py

```diff
@@ -143,7 +130,7 @@ def coefficient(ufl_coefficient, r):
                              point_index=quadrature_index,
                              coefficient=coefficient,
                              index_cache=index_cache)
-        if parameters["unroll_indexsum"]:
+        if parameters.get("unroll_indexsum"):
```

Comment:
I guess we could just move the sanitization of parameters to `compile_integral` from `compile_form`, then everything would be here when expected.